app/UiTPAS/UiTPASServiceProvider.php

```diff
@@ -71,6 +72,15 @@ public function register(): void
             );
         });
 
+        $this->app->singleton(ActivateUdbOrganizer::class, function () {
+            return new ActivateUdbOrganizer(
+                $this->app->get(UdbOrganizerRepository::class),
+                $this->app->get(IntegrationRepository::class),
+                $this->app->get(UiTPASApiInterface::class),
+                ClientCredentialsContextFactory::getUitIdProdContext(),
+            );
+        });
+
         if (!config(UiTPASConfig::AUTOMATIC_PERMISSIONS_ENABLED->value)) {
```

Comment:
This is a very weird approach to disable `bootstrapEventHandling`

Reply:
I assume you want the if inside the function? I will move it